.github/scale-config.yml

```diff
@@ -220,3 +220,27 @@ runner_types:
     instance_type: g5.4xlarge
     is_ephemeral: false
     os: windows
+  linux.2xlarge.memory:
+    disk_size: 200
+    instance_type: r5.2xlarge
+    is_ephemeral: false
+    os: linux
+    ami: al2023-ami-2023.5.202*-kernel-6.1-x86_64
```

Comment:
What is the size of the RAM for a memory intensive runner?

Reply:
Here you go https://aws.amazon.com/ec2/instance-types/r5/ (comparing with the current c5 instances you are using https://aws.amazon.com/ec2/instance-types/c5/), let me also draft a post to announce this to a broader audience with these information.

Reply:
Yes, please do post this new runner instance. In ExecuTorch we should start replacing the expensive runners with this new type of runner for most use-cases, and should also guide the developers to always try with this memory intensive runner first before going with a 12x runner or so (to check the workload is memory bound or cpu bound)